lang/go/eval/symbol/named.go

```diff
@@ -33,6 +33,12 @@ type NamedSymbol struct {
 
 var _ Symbol = &NamedSymbol{}
 
+// DisassembleToGo converts a Ko value into a Go value
+func (named *NamedSymbol) DisassembleToGo(span *model.Span) (reflect.Value, error) {
+	return DeconstructKind(span, named.Value).DisassembleToGo(span)
```

Comment:
A NamedSymbol holds Go values that are named, e.g. 
type MyStruct struct { ... } is named, whereas
struct { ... } is not named.

The call to DeconstructKind strips the name. This is not necessary here.
Just use: return name.Value.Interface()